packages/desktop/views/dashboard/send-flow/views/SelectRecipientView.svelte

```diff
@@ -54,8 +60,11 @@
                 recipient: undefined,
             }
         })
-
-        selectorOptions = [mainNetworkOption, ...iscpNetworkChainsOptions]
+        if (canTransferAssetToLayer1(($newTransactionDetails as NewTokenTransactionDetails).asset)) {
+            selectorOptions = [layer1NetworkOption, ...iscpNetworkChainsOptions]
+        } else {
+            selectorOptions = iscpNetworkChainsOptions
```

Comment:
im not sure about which direction are possible at all.
can i send an IRC30 or ERC20 from e.g. shimmerEVM to shimmerEVM2?

in my understanding, no. Therefore we should only include the following:
```
        if (canTransferAssetToLayer1(($newTransactionDetails as NewTokenTransactionDetails).asset)) {
            selectorOptions = [layer1NetworkOption, chainOptions[$newTransactionDetails.asset.chainId] ]
        } else {
            selectorOptions = chainOptions[$newTransactionDetails.asset.chainId] 
```

Is that right @Tuditi

Reply:
The protocol has been designed so that it can handle bridges across different layer 2's easily. However, there is not that much documentation/examples yet, so for now, it might make sense to only allow sending on the same layer. 

We haven't designed L2a <> L2b from a technical perspective, so it might be safer to go with Mark's suggestion.

Reply:
Yes, that's only valid for IRC-30. ERC-20 can only be transferred inside the same network.